pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -218,6 +218,8 @@ type AWSCloudConfig struct {
 		Zone string
 
 		KubernetesClusterTag string
+
+		DisableSecurityGroupIngress bool
```

Comment:
Can we have a comment about what this is for please.  The comment in the PR is great (or an abridged version of it)